spec/features/social_features_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+feature "Social networking" do
+    given(:jane) { Fabricate(:user) }
+    given(:joe) { Fabricate(:user) }
+    given(:tv_comedies) { Fabricate(:category, name: 'TV Comedies') }
+
+  scenario "following a user" do 
+    futurama = Fabricate(:video, title: 'Futurama', category: tv_comedies)
+    jane_review = Fabricate(:review, video: futurama, user: jane)
+    
+    sign_in(joe)
+    expect(page).to have_content "Welcome, #{joe.full_name}"
```

Comment:
Thanks! I removed the expect(page) assertions all together. So, to be clear - I should stay away from using expect(page) to assert the presence of something I could potentially change in the future, right? However, does it make sense to use it to verify presence of of an element that should always be present (Ex, when "following" a user, that user should always be present on the "people I follow" page)? I want to make sure I'm using it properly. thx